snow/networking/handler/health.go

```diff
@@ -47,9 +47,14 @@ func (h *handler) networkHealthCheck() (interface{}, error) {
 		"disconnectedValidators": h.getDisconnectedValidators(),
 	}
 
-	var err error
 	subnetConfig := h.subnet.Config()
-	minPercentConnected := subnetConfig.ConsensusParameters.MinPercentConnectedHealthy()
+
+	if subnetConfig.ConsensusParameters.SimplexParams != nil {
+		return details, nil
```

Comment:
why would we return here though? Don't we want to report unhealthy if we're not connected to sufficient stake? 

The stake criteria in our case should be different than the stake criteria of snowman, it should be 33% instead of `α / K` as in snowman, but still I think this is worth retaining with a different threshold. 

Thoughts?